extensions/wikia/TemplateDraft/TemplateDraftHelper.class.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+class TemplateDraftHelper {
+
+	/**
+	 * Checks if the Title object is a Draft subpage of a template
+	 *
+	 * @param Title $title
+	 * @return bool
+	 */
+	public function isTitleDraft( Title $title ) {
+		/**
+		 * TODO: Improve this check (i18n)
+		 */
+		return strpos( $title->getText(), '/Draft' ) !== false;
```

Comment:
This is a bit loose as it doesn't specify where in the string '/Draft' where we only want it if it's the end (i.e. it's the current subpage). Something like `Foo/Draft 2012` would return true here.